package.json

```diff
@@ -28,7 +28,8 @@
     "del": "~1.1.0",
     "vinyl-buffer": "~1.0.0",
     "vinyl-source-stream": "~1.0.0",
-    "babelify": "~5.0.4"
+    "babelify": "~5.0.4",
```

Comment:
Hum, I've updated the code since this as babelify is now in version 6.0.2, so you might need to rebase this PR.